src/vs/workbench/services/languageRuntime/common/languageRuntimeDataExplorerClient.ts

```diff
@@ -122,6 +125,11 @@ export class DataExplorerClientInstance extends Disposable {
 	 */
 	public status: DataExplorerClientStatus = DataExplorerClientStatus.Idle;
 
+	/**
+	 * The guessed code syntax for the data explorer instance, if we generate code.
+	 */
+	public guessedSyntax: CodeSyntax = { code_syntax: 'No available syntaxes' };
```

Comment:
I say more about this below, but having a special string indicate that this piece of information is unavailable doesn't feel right.